android/build.gradle

```
@@ -33,6 +33,10 @@ def useDedicatedExecutor = rootProject.hasProperty('AsyncStorage_dedicatedExecut
         ? rootProject.properties['AsyncStorage_dedicatedExecutor']
         : false
 
+def disableLocalizedCollators = rootProject.hasProperty('AsyncStorage_noLocalizedCollators')
+        ? rootProject.properties['AsyncStorage_noLocalizedCollators']
+        : false
```

Nit: We don't really need the ternary but I see we've done the same above...
```suggestion
def disableLocalizedCollators = rootProject.hasProperty('AsyncStorage_noLocalizedCollators')
        && rootProject.properties['AsyncStorage_noLocalizedCollators']
```

---

Yes, I kept it for consistency